appinventor/components/src/com/google/appinventor/components/runtime/Direction.java

```diff
@@ -0,0 +1,25 @@
+
+// TODO: Not sure which package these should go in.
+package com.google.appinventor.components.runtime;
+
+public enum Direction {
+    North(1),
```

Comment:
Code style: 2 space indent for Google Java codestyle. You should also be able to run `ant checkstyle` to get feedback before committing.